webui/react/src/routes/index.ts

```diff
@@ -13,27 +14,29 @@ import { ensureAbsolutePath, isFullPath, parseUrl } from 'utils/routes';
  * meaning React will attempt to load the path outside of the internal routing
  * mechanism.
  */
-export interface RouteConfigItem extends RouteProps {
+export interface RouteConfig extends RouteProps {
   id: string;
   icon?: string;
   path: string;
   popout?: boolean;
+  redirect?: string;
   suffixIcon?: string;
-  title: string;
+  title?: string;
   needAuth?: boolean;
 }
 
+const defaultPath = '/det/dashboard';
```

Comment:
suggestion: use the one from line 63 `defaultAppRoute = appRoutes[0];`

Reply:
@hamidzr  hmmm, how do you avoid the circular variable dependencies?

Reply:
one way I think would be to directly set `defaultAppRoute` to `dashboardRoute` instead of indexing into the `appRoutes` list